src/likelihoods/gamma.jl

```diff
@@ -1,21 +1,24 @@
 """
-    GammaLikelihood(α)
+    GammaLikelihood(α::Real=1.0, l::AbstractLink=ExpLink())
 
 Gamma likelihood with fixed shape `α`.
 
 ```math
-    p(y|f) = Gamma(y | α, θ=exp(f))
+    p(y|f) = Gamma(y | α, θ=l(f))
```

Comment:
I'm not sure that having the \theta character here doesn't create more confusion than clarity. Below is states that `l(f)` is the scale, which I would imagine is sufficient?